get/releases.go

```diff
@@ -0,0 +1,94 @@
+package get
+
+import (
+	"context"
+	"os"
+	"sort"
+	"strconv"
+	"text/tabwriter"
+
+	apps "github.com/ninech/apis/apps/v1alpha1"
+	"github.com/ninech/nctl/api"
+	"github.com/ninech/nctl/api/util"
+)
+
+type releasesCmd struct {
+	Name string `arg:"" help:"Name of the Application for which to get the Releases. If omitted all Applications will be listed." default:""`
+}
+
+func (cmd *releasesCmd) Run(ctx context.Context, client *api.Client, get *Cmd) error {
+	releaseList := &apps.ReleaseList{}
+
+	if err := get.list(ctx, client, releaseList, matchApplicationName(cmd.Name)); err != nil {
+		return err
+	}
+
+	if len(releaseList.Items) == 0 {
+		printEmptyMessage(apps.ReleaseKind, client.Namespace)
+		return nil
+	}
+
+	switch get.Output {
+	case full:
+		return printReleases(releaseList.Items, get, true)
+	case noHeader:
+		return printReleases(releaseList.Items, get, false)
+	}
+
+	return nil
+}
+
+// prepareReleasesOutput prepares Release collection for printing
+func prepareReleasesOutput(releases []apps.Release) map[string][]apps.Release {
+	orderedReleases := make(map[string][]apps.Release)
+
+	for _, release := range releases {
+		orderedReleases[release.ObjectMeta.Labels[util.ApplicationNameLabel]] =
+			append(orderedReleases[release.ObjectMeta.Labels[util.ApplicationNameLabel]], release)
+	}
+
+	for _, v := range orderedReleases {
+		sort.Slice(v, func(i, j int) bool {
+			return v[i].CreationTimestampNano < v[j].CreationTimestampNano
+		})
+	}
+
+	return orderedReleases
+}
+
+func printReleases(releases []apps.Release, get *Cmd, header bool) error {
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 4, ' ', 0)
+
+	if header {
+		get.writeHeader(
+			w,
+			"BUILDNAME",
+			"NAME",
+			"APPLICATION",
+			"SIZE",
+			"REPLICAS",
+			"STATUS",
+			"AGE",
+		)
+	}
+
+	orderedReleases := prepareReleasesOutput(releases)
+
+	for _, orderedRelease := range orderedReleases {
+		for _, r := range orderedRelease {
+			get.writeTabRow(
+				w,
+				r.ObjectMeta.Namespace,
+				r.Spec.ForProvider.Build.Name,
+				r.ObjectMeta.Name,
+				r.ObjectMeta.Labels[util.ApplicationNameLabel],
+				string(*r.Spec.ForProvider.Config.Size),
+				strconv.Itoa(int(*r.Spec.ForProvider.Config.Replicas)),
+				string(r.Status.AtProvider.ReleaseStatus),
+				r.ObjectMeta.CreationTimestamp.String(),
```

Comment:
Instead of printing the whole timestamp, can we display the relative age of the release? I already did this in the build using `duration.HumanDuration(time.Since(build.CreationTimestamp.Time)))`. It's the same function `kubectl` calls to populate the age column.